website/docs/docs/core/installation-overview.md

```diff
@@ -52,4 +52,7 @@ Most command-line tools, including dbt, have a `--help` flag that you can use to
 :::
 
 ## Related docs
```

Comment:
I completely understand that we're trying to encourage people to move from Core to Cloud, but putting these migration guides at the bottom of the pages for installing Core and calling them "Related docs" feels a little disingenuous. Maybe we could retitle them something like "Moving from Core to Cloud" or "Upgrading your environment" I'm open to other titles, but calling them "Related docs" right after they get done installing Core feels like we're letting them down. Related docs should lead to more Core workflows.

Reply:
for sure, makes sense to retitle them to 'upgrading your environment'. was trying to  help the guides be more discoverable since we currently have 35 guides.